Kitodo/src/test/java/org/kitodo/selenium/testframework/Browser.java

```diff
@@ -93,7 +93,7 @@ private static void provideChromeDriver() throws IOException {
 
         if (!driverFile.exists()) {
             logger.debug(driverFile.getAbsolutePath() + " does not exist, providing chrome driver now");
-            WebDriverProvider.provideChromeDriver(CHROME_DRIVER_VERSION, DOWNLOAD_DIR, DRIVER_DIR);
+            WebDriverProvider.provideChromeDriver(DOWNLOAD_DIR, DRIVER_DIR);
```

Comment:
If you remove usage of CHROME_DRIVER_VERSION, remove also declaration of this variable (line 48).